### DISCUSSION TITLE
Sound device for effects

### DISCUSSION THREAD

**Discussion by Yamh631** [awawa-dev/HyperHDR#480] [Q&A] [answered]
Hello,

I had a question about "Sound device for effects".
Is it also used to adjust the intensity of the LEDs when watching a film?

![Capture d’écran 2023-01-07 215810](https://user-images.githubusercontent.com/35656304/211170080-bd17cde0-4197-47da-8b22-3a459cb2af14.png)

Or this function is only used when applying an effect with the remote control:

![Capture d’écran 2023-01-07 220021](https://user-images.githubusercontent.com/35656304/211170145-35003589-b6f4-4f8d-85d8-90916fff3681.png)

Thanks :)

**Comment by awawa-dev** [comment, marked as answer]
Hi
It works only as a effect. If you want to have it working when you watch TV, you need to create a separate instance that is using that effect and provide LED strip or lamp for it (different than you use e.g. for the TV).

**Reply by Yamh631** [reply]
Thanks :)
